plugins/modules/mso_schema_template_bd.py

```diff
@@ -239,6 +304,11 @@ def main():
         vrf=dict(type='dict', options=mso_reference_spec()),
         dhcp_policy=dict(type='dict', options=mso_dhcp_spec()),
         subnets=dict(type='list', elements='dict', options=mso_subnet_spec()),
+        unknown_multicast_flooding=dict(type='str', choices=['optimized_flooding', 'flood']),
+        multi_destination_flooding=dict(type='str', choices=['flood_in_bd', 'drop']),
+        ipv6_unknown_multicast_flooding=dict(type='str', choices=['optimized_flooding', 'flood']),
+        arp_flooding=dict(type='bool'),
```

Comment:
Is there an default value for arp_flooding? If yes do we need mention it in line 310.

Reply:
yes we need to mention it because there's no default. Besides we need to declare it regardless, if we want to explicitly set it as true.